test/v1/services/AmendOtherReliefsServiceSpec.scala

```diff
@@ -99,7 +99,15 @@ class AmendOtherReliefsServiceSpec extends UnitSpec {
         "SERVICE_UNAVAILABLE"              -> InternalError
       )
 
-      input.foreach(args => (serviceError _).tupled(args))
+      val tysErrors = Seq(
+        "INVALID_CORRELATION_ID" -> InternalError,
+        "INVALID_TAX_YEAR"       -> TaxYearFormatError,
+        "INVALID_PAYLOAD"        -> InternalError,
+        "TAX_YEAR_NOT_SUPPORTED" -> RuleTaxYearNotSupportedError,
+        "UNPROCESSABLE_ENTITY"   -> InternalError
+      )
+
+      (errors ++ tysErrors).foreach(args => (serviceError _).tupled(args))
     }
   }
```

Comment:
ideally we should also move the test trait to the bottom of the spec so that the tests are the first + main thing visible when viewing the file (could move all the vals i.e. taxYear, nino, correlationId, body, request data into the test trait too so it is all out of the way and contained)